[sarthak181] iP

sarthak181 opened the pull request.

Commit:
Add Gradle support

sarthak181 committed:
Level 1

sarthak181 committed:
level 3

sarthak181 committed:
level 4

sarthak181 committed:
level 5

sarthak181 committed:
level 6

wz2k commented on src/main/java/Duke.java:
```diff
@@ -6,5 +8,104 @@ public static void main(String[] args) {
+                case 5:
+                    System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+
+            }
```
cases in switch statements should not be indented. eg
```java
switch (variable) {
case 0:
    return 0;
case 1:
    return 1;
}

wz2k commented on src/main/java/Duke.java:
```diff
@@ -6,5 +8,104 @@ public static void main(String[] args) {
+        if (str.startsWith("unmark ")) return 1;
+        if (str.startsWith("todo ")) return 2;
+        if (str.startsWith("deadline ")) return 3;
+        if (str.startsWith("event ")) return 4;
```
Use K&R style brackets. eg
```java
if (condition) {
    return 0;
}
```
I suggest just using the string directly in the switch statement but this is entirely up to you.

wz2k commented on src/main/java/Duke.java:
```diff
@@ -1,3 +1,5 @@
+import java.util.*;
+
 public class Duke {
```
Classes should be placed in a package. You can just place all your current class in a duke package minimally

wz2k commented on src/main/java/Event.java:
```diff
@@ -0,0 +1,19 @@
+    protected String from;
+    protected String to;
+
+    public Event(String description, String from, String to) throws DukeException{
```
'DukeException{'. Missed out on a space between n and {.

wz2k commented on src/main/java/Duke.java:
```diff
@@ -1,3 +1,5 @@
+import java.util.*;
```
Wildcard imports should not be used. Imported classes should be listed explicitly

wz2k reviewed (commented):
Good code overall besides a few changes that should be made to follow our module's coding standard. Do remember to add descriptive header comments (eg. javadocs) to your public classes and methods. Keep up the good work! :thumbsup:

sarthak181 committed:
level 7

sarthak181 committed:
level 8

sarthak181 committed:
Merge tag 'branch-Level-8'

sarthak181 committed:
A-MoreOOP

sarthak181 committed:
A-Packages

sarthak181 committed:
Merge branch 'add-gradle-support'

sarthak181 committed:
Added JUnit

sarthak181 committed:
Level 9

sarthak181 committed:
Added javadoc comments

sarthak181 committed:
amended

sarthak181 committed:
Merge branch 'branch-A-CodingStandard'

sarthak181 committed:
Merge branch 'branch-A-Level-9'

# Conflicts:
#	src/main/java/duke/TaskList.java

sarthak181 committed:
Updated Readme

sarthak181 committed:
Complete level 10

sarthak181 committed:
added checkstyle files

sarthak181 committed:
Merge branch 'A-correct-CheckStyle'

sarthak181 committed:
Added Assertions

sarthak181 committed:
Merge pull request #2 from sarthak181/branch-A-Assertions

Merge Added Assertions

sarthak181 committed:
Amended Bugs in Code

Changed the way that the code responds when an exception is caught.
Changed it so that when an exception is caught, the Duke bot actually
tells the user that there is an error.

sarthak181 committed:
Added help command

This help command is simply the user typing help in all lowercase and
hitting enter.
The commands aren't too easy to remember, so this help command will
make it easier to use the app.

sarthak181 committed:
Made duke and user a circle

sarthak181 committed:
Merge branch 'branch-C-Help'

sarthak181 committed:
Amended Bugs

sarthak181 committed:
Amended Bugs

sarthak181 committed:
Finalize Ui.png

sarthak181 committed:
a

sarthak181 committed:
Merge branch 'master' of https://github.com/sarthak181/ip

sarthak181 committed:
delete Ui.PNG

sarthak181 committed:
finalize
